pkg/windows/windows.go

```diff
@@ -591,6 +597,21 @@ func (vm *windows) createDirectories() error {
 // removeDirectories removes all directories created as part of the configuration process
 func (vm *windows) removeDirectories() error {
 	vm.log.Info("removing directories")
+	existingFiles, err := getFilesToTransfer()
+	if err != nil {
+		return errors.Wrapf(err, "error getting list of files transferred to the instance")
+	}
+	for src, dest := range existingFiles {
+		filePath := dest + "\\" + filepath.Base(src.Path)
+		if out, err := vm.Run(rmFileCmd(filePath), true); err != nil {
+			return fmt.Errorf("unable to remove file %s, out: %s, err: %s", filePath, out, err)
+		}
+	}
+	for _, file := range wicdFiles {
+		if out, err := vm.Run(rmFileCmd(file), true); err != nil {
+			vm.log.V(1).Error(err, "unable to remove file", "file", file, "out", out)
+		}
+	}
 	for _, dir := range RequiredDirectories {
 		if out, err := vm.Run(rmDirCmd(dir), true); err != nil {
 			return fmt.Errorf("unable to remove directory %s, out: %s, err: %s", dir, out, err)
```

Comment:
This actually does not fix the issue. If there is an error at this stage, we will attempt to reconcile again and fail as the WICD files have been removed. You don't need to delete the WICD files individually but just make the directory deletion best effort.

Reply:
Agreed directory deletion should be best effort. I am deleting WICD files separately as a best effort as the cleanup command requires all the three files, if any one of them is deleted but the other one fails then we would end up in the same spot.

Reply:
If you are going to make WICD file deletion and directory deletion all best effort, what is the point of deleting WICD files individually? We should be trying to reduce the number of SSH calls we make.

Reply:
ack, they will get removed as part of the directory deletion.